cluster/gce/gci/flexvolume_node_setup.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sets up FlexVolume drivers on GCE COS instances using mounting utilities packaged in a Google
+# Container Registry image.
+# The user-provided FlexVolume driver(s) must be under /flexvolume of the image filesystem.
+# For example, the driver k8s/nfs must be located at /flexvolume/k8s~nfs/nfs .
+#
+# This script is intended to be used on an instance with no FlexVolume installed.
+# Must be executed under /home/kubernetes/bin with sudo.
+# Warning: kubelet will be restarted upon successful execution.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+MOUNTER_IMAGE=${1:-}
+MOUNTER_PATH=/home/kubernetes/flexvolume_mounter
+VOLUME_PLUGIN_DIR=/home/kubernetes/flexvolume
```

Comment:
Can't seem to access it as either user or root. Might be because the root login session is different from when the instance is set up?